### ISSUE TITLE
TypeError: 'type' object is not subscriptable

### ISSUE THREAD

**Issue by mubix** [ly4k/Certipy#22]
When running certipy I'm getting the following:
```bash
# certipy
Traceback (most recent call last):
  File "/usr/local/bin/certipy", line 11, in <module>
    load_entry_point('Certipy==2.0', 'console_scripts', 'certipy')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 490, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2854, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2445, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2451, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.8/dist-packages/Certipy-2.0-py3.8.egg/certipy/entry.py", line 8, in <module>
    from certipy import auth, ca, find, forge, relay, request, shadow, template, version
  File "/usr/local/lib/python3.8/dist-packages/Certipy-2.0-py3.8.egg/certipy/shadow.py", line 22, in <module>
    class Shadow:
  File "/usr/local/lib/python3.8/dist-packages/Certipy-2.0-py3.8.egg/certipy/shadow.py", line 54, in Shadow
    def get_key_credentials(self, target_dn: str, user: LDAPEntry) -> list[bytes]:
TypeError: 'type' object is not subscriptable

```

**Comment by mubix** [comment]
Information about my environment:
#### Python Version
- Python 3.8.10 (default, Nov 26 2021, 20:14:08)

#### Installation Method
- git commit version 447208db6ac8eee3bf92abd37999f8a362796f6b

**Comment by twigatech** [comment]
Requires Python3.9

**Comment by ly4k** [comment]
Thank you to both of you. This is actually a mistake from my side. It should have been -> List and not -> list. I will fix it later today

**Comment by ly4k** [comment]
Should be fixed in version 2.0.1.
